docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Also looks good. You could consider giving one context region (either 1,2 or 5,6) significantly more samples than the other so that it also biases the population model away from the region that we're interpolating.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>